torch/_torch_docs.py

```diff
@@ -2360,12 +2360,20 @@ def merge_dicts(*dicts):
 corresponding to the polar coordinates with absolute value :attr:`abs` and angle
 :attr:`angle`.
 
+.. note::
+    This implementation has the same exact logic as C++'s implementation of
+    `std::polar <https://en.cppreference.com/w/cpp/numeric/complex/polar>`_
+    which is different from Python's `cmath.polar` and SciPy's `linalg.polar`.
+    The latter implementations in `cmath` and `linalg` perform the inverse operation
+    (i.e. take in a complex number as input and return a pair of absolute value and angle as output).
+
 .. math::
     \text{out} = \text{abs} \cdot \cos(\text{angle}) + \text{abs} \cdot \sin(\text{angle}) \cdot j
 """ + r"""
 Args:
-    abs (Tensor): The absolute value the complex tensor. Must be float or
-        double.
+    abs (Tensor): The absolute value the complex tensor. Must be float or double.
+        Note: this input is not validated, and the function will still perform the computation as described
```

Comment:
let's move the extra detail to the function description above.

Reply:
Done